src/main/resources/hudson/plugins/timestamper/annotator/TimestampAnnotatorFactory3/settingsForm.jelly

```diff
@@ -0,0 +1,20 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core">
+<form>
+    <label>
+        <input type="radio" name="timestamps" id="timestamper-systemTime">${%System clock time}</input>
+    </label>
+    <br/>

```

Comment:
can this be handled with css rather than a br?

Reply:
@timja good point, done in the last commit.